### PR TITLE
Docs: move and update gaia client instructions and remove deploy-testnets.md

### PR DESCRIPTION
**Pull request by nooomski**
This PR moves the gaia client instructions to the Hub Tutorials folder and adds information about the keyring backend. The outdated deploy-testnets is removed in favor of the [testnets repo](https://github.com/cosmos/testnets/).

### CODE REVIEW

File: docs/hub-tutorials/join-testnet.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-order: 4
+order: 3
 -->
 
 # Join the Public Testnet
 
 | Current Upgrade | Chain Id       | Upgrade Block Height | Upgrade Date     |
 | --------------- | -------------- | -------------------- | ---------------- |
-| Theta           | `theta-testnet-001` | 9283650   | March 17 2021 |
+| Theta           | `theta-testnet-001` | TBD   | March 17 2021 |
```
Comment (glnro) [inline docs/hub-tutorials/join-testnet.md]: Why is this tbd now?

File: docs/hub-tutorials/join-testnet.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
 
 ## Background
-The current Cosmos Hub Testnet is running on the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
+The current Cosmos Hub Testnet is running to prepare for the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
```
Comment (glnro) [inline docs/hub-tutorials/join-testnet.md]: The theta upgrade already happened...do you mean Rho?

File: docs/hub-tutorials/join-testnet.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 
 For those who just need instructions on performing the upgrade, see the [Upgrade](#upgrading) section.
 
 ## Releases
-If syncing before the Theta update, checkout [`v6.0.0`](https://github.com/cosmos/gaia/tree/v6.0.0). **If syncing after the upgrade, see [`v7.0.0-rc0`](https://github.com/cosmos/gaia/tree/v7.0.0-rc0).**
+If syncing before the Theta update, checkout [`v6.0.0`](https://github.com/cosmos/gaia/tree/v6.0.0). Until a release is cut for the upgrade, feel free to track the [`theta-prepare` branch](https://github.com/cosmos/gaia/tree/theta-prepare).
```
Comment (glnro) [inline docs/hub-tutorials/join-testnet.md:18]: We can remove this actually, and just tell people to checkout the `v7.0.0-rc` to sync

File: docs/hub-tutorials/join-testnet.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
 
 ## Prerequisites
 
 **Hardware**
 
 It's recommended that public testnet nodes are running on machines with at least `16GB` of RAM.
 
-**Make sure Go & Gaia are [properly installed](../getting-started/installation.md). The most recent Gaia version for the Theta Testnet is [`v7.0.0-rc0`](https://github.com/cosmos/gaia/tree/v7.0.0-rc0).**
+**Make sure Go & Gaia are [properly installed](../getting-started/installation.md). The most recent Gaia version for the Theta Testnet is [`v6.0.0`](https://github.com/cosmos/gaia/tree/v6.0.0).**
```
Comment (glnro) [inline docs/hub-tutorials/join-testnet.md]: This is wrong

**Review by glnro — CHANGES_REQUESTED** [review]

File: docs/hub-tutorials/join-testnet.md (hunk 4 above)
Comment (glnro) [reply docs/hub-tutorials/join-testnet.md]: Curious why this was changed?

File: docs/hub-tutorials/join-testnet.md (hunk 1 above)
Comment (okwme) [reply docs/hub-tutorials/join-testnet.md]:
```suggestion
| Theta           | `theta-testnet-001` | 9283650   | March 17 2021 |
```

File: docs/hub-tutorials/join-testnet.md (hunk 2 above)
Comment (okwme) [reply docs/hub-tutorials/join-testnet.md]:
```suggestion
The current Cosmos Hub Testnet is running on the [Theta Upgrade](https://interchain-io.medium.com/preparing-for-the-cosmos-hub-v7-theta-upgrade-2fc41ce34787). Visit the [testnet explorer](https://explorer.theta-testnet.polypore.xyz/) to view all on chain activity.
```

File: docs/hub-tutorials/join-testnet.md (hunk 4 above)
Comment (okwme) [reply docs/hub-tutorials/join-testnet.md]:
```suggestion
**Make sure Go & Gaia are [properly installed](../getting-started/installation.md). The most recent Gaia version for the Theta Testnet is [`v7.0.0-rc0`](https://github.com/cosmos/gaia/tree/v7.0.0-rc0).**
```

File: docs/hub-tutorials/join-testnet.md (hunk 3 above)
Comment (okwme) [reply docs/hub-tutorials/join-testnet.md:18]: you want to tweak this section @glnro and we can merge?

**Comment by nooomski** [comment]
Updated the broken links. Anything else that still needs to be done here?

**Comment by nooomski** [comment]
Not quite sure why the https://explorer.theta-testnet.polypore.xyz/  link keeps throwing an error. It's a valid link 🤔

File: docs/hub-tutorials/join-testnet.md (hunk 3 above)
Comment (glnro) [reply docs/hub-tutorials/join-testnet.md:18]: Sorry I missed this, sounds good.

**Review by glnro — APPROVED** [review]

**Pull request merged**